### DISCUSSION TITLE
question: do custom webpack aliases work during SSR with next?

### DISCUSSION THREAD

**Discussion by cdaringe** [vercel/next.js#13122] [Help]
# question

supposi i have a component:

```js
import getBananas from "@best-lib/fruit"
export const Fruit => <pre>{getBananas()}</pre>
```

and i'm resolving `@best-lib/fruit` through a webpack alias, should i expect my server to also be able to resolve `@best-lib/fruit`? in other words, does server code use webpack aliases, or should i be using babel import transforms? alternate angle on the sasme consideration, what if i'm writing an API route in next, and i want a `getBananas` resolved in both an API route AND a UI route--will webpack aliases do all the good stuff?

**Comment by jamesmosier** [comment]
Yes, webpack aliases will work in both your client & server (API routes, getStaticProps, getServerProps, etc.) code 🎉 

Also (and you may know this!) Next can now do these aliases for you: https://nextjs.org/docs/advanced-features/module-path-aliases
